osbs/build/build_request.py

```diff
@@ -697,7 +697,7 @@ def render_resolve_composes(self):
             return
 
         odcs_url = self.spec.odcs_url.value
-        if not odcs_url:
+        if not odcs_url or self.spec.yum_repourls.value:
```

Comment:
Lets have a log message about why the plugin was removed

Reply:
ok